pliers/extractors/api/clarifai.py

```diff
@@ -97,8 +97,16 @@ def _query_api(self, objects):
 
     def _parse_annotations(self, annotation):
         data_dict = {}
-        for tag in annotation['data']['concepts']:
-            data_dict[tag['name']] = tag['value']
+        # check whether the model is the face detection model
+        if self.model_name == 'a403429f2ddf4b49b307e318f00e528b':
```

Comment:
Do we know that this is a static label for the face model? I'm worried that this is just a hash they use to encode the specific version of the model, and that it might change frequently. It might be better to rely on the model name stored in `self`.